.github/workflows/publish-npm-packages.yml

```
@@ -125,13 +127,15 @@ jobs:
       - name: Tag package as latest for Stable releases
         if: github.event.inputs.tag == 'latest'
         run: |
-          NPM_PACKAGE=$(jq -r .name ./extensions/${{ matrix.extension }}/${{ matrix.version }}/package.json)
-          NPM_VERSION=$(jq -r .version ./extensions/${{ matrix.extension }}/${{ matrix.version }}/package.json)
+          cd $(node ${{ env.SCRIPTS_DIR }}/build-npm-binaries.js ${{ matrix.extension }} ${{ matrix.version }})
```

wait should this be?
```suggestion
          cd $(node ${{ env.SCRIPTS_DIR }}/get-package-dir.js
```